Yakkity-Yulelogs/reactive-nutshell
# docs: add demo and update README instructions

# Description

Update README.md to include demo and registration/login instructions

Fixes #71 

## Type of change

- [x] This change is ONLY a documentation update

# Testing Instructions

`git fetch --all`
`git checkout rb-readme`
Open the PREVIEW mode of the README and check out the new demo GIF and added instructions. 
Check for any spelling/grammar errors PLEASE

# Checklist:
